metrics/README.md

```diff
@@ -192,6 +193,20 @@ Add a fully formed JSON fragment to the JSON array store.
 | ---- | ----------- |
 | json | A fully formed JSON fragment |
 
+#### metrics_json_add_array_fragment(json)
+
+Add a fully formed JSON fragment to the current array element.
+
+| Arg  | Description |
+| ---- | ----------- |
+| json | A fully formed JSON fragment |
+
+#### metrics_json_close_array_element()
+
+Finalize (close) the current array element. This will encorporate
```

Comment:
Lines 206-208 suggested rewrite: 
... This incorporates any array_fragment parts into the current array element, closes that array element, and resets the in-flight array_fragment store.